tests/auth/registration/registration.spec.js

```diff
@@ -0,0 +1,170 @@
+import {expect, test} from '@playwright/test';
+
+test.describe('Registration', () => {
+
+    async function openRegistrationPopup(page) {
+        return await test.step('Open registration popup', async () => {
```

Comment:
It is good solution to wrap some logic to steps but I would wrap it in the test. It is not objection but advice

Reply:
but you can leave it as is, it is also not so bad